lib/signs_ui/signs/expiration.ex

```diff
@@ -0,0 +1,76 @@
+defmodule SignsUI.Signs.Expiration do
+  @moduledoc """
+  Handle cleaning up settings in the configuration that have since expired
+  """
+
+  @type state :: %{
+          time_fetcher: (() -> DateTime.t()),
+          loop_ms: integer(),
+          sign_state: pid()
+        }
+
+  @spec start_link(Keyword.t()) :: GenServer.on_start()
+  def start_link(opts \\ []) do
+    name = opts[:name] || __MODULE__
```

Comment:
I know we haven't been very good about it elsewhere, but since `opts` is a `Keyword.t` it might be better here to work with it through the `Keyword` module, in particular `Keyword.get/3`.

Reply:
Good point, I'll change it.